test/jdk/javax/swing/JTable/8236907/LastVisibleRow.java

```diff
@@ -75,9 +77,12 @@ public void run() {
             clearSelect();
             testRobot.waitForIdle();
             BufferedImage bufferedImageAfter = testRobot.createScreenCapture(getCaptureRect());
+
             testRobot.delay(1000);
 
             if (!compare(bufferedImageBefore, bufferedImageAfter)) {
+                ImageIO.write(bufferedImageBefore, "png", new File("bufferedImageBefore.png"));
+                ImageIO.write(bufferedImageAfter, "png", new File("bufferedImageAfter.png"));
                 throw new RuntimeException("Test Case Failed!!");
```

Comment:
Probably you can change the failure message also now which I guess should be something like "Last row is not visible" 
Also, I see from the screen.png artifacts generated when the test failed shows a empty desktop with "Home" "Trash" icons on lower right of screen so not sure if this generated images also will help but let's try..

Reply:
Done.....